BookingsApi/BookingsApi.DAL/Queries/GetBookingsByCaseTypesQuery.cs

```diff
@@ -59,15 +59,12 @@ public GetBookingsByCaseTypesQueryHandler(BookingsDbContext context)
                 .Include(x => x.HearingVenue)
                 .AsNoTracking();
 
-            if (query.CaseTypes.Any())
-            {
-                var dayToday = DateTime.UtcNow;
-                var dateNow = new DateTime(dayToday.Year, dayToday.Month, dayToday.Day);
-                hearings = hearings.Where(x =>
-                    x.ScheduledDateTime > dateNow && query.CaseTypes.Contains(x.CaseTypeId));
-            }
+            if (query.CaseTypes.Any()) hearings = hearings.Where(x => query.CaseTypes.Contains(x.CaseTypeId));
```

Comment:
Could you put this inside curly brackets. I find it more readable and it's more maintainable if changes need to be made. Up to you though, I don't think we have any style guidelines around that.

Reply:
Yeah will do